src/libs/actions/Task.js

```diff
@@ -791,6 +791,18 @@ function cancelTask(taskReportID, taskTitle, originalStateNum, originalStatusNum
                 [optimisticReportActionID]: optimisticCancelReportAction,
             },
         },
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT}${taskReportID}`,
+            value: {
+                hasDraft: false,
```

Comment:
@hungvu193 Can we add this hasDraft value [here](https://github.com/Expensify/App/pull/26547/files#diff-941175f813fc9ba999d5f8221c16657a8f274cc7a803657578d5e4f01ae26da4L784)?